### ISSUE TITLE
Dollar sign in root reducer and master middleware when generating cycle

### ISSUE THREAD

**Issue by paul-codes-1** [amattson21/eos-redux#20]
Generated a cycle just now and here is what I got in root reducer and master middleware.

``` javascript
import { combineReducers } from 'redux';
  import CarsReducer from './cars_reducer';

const RootReducer = combineReducers({

$     cars: CarsReducer
});

export default RootReducer;
```

``` javascript
import { applyMiddleware } from 'redux';
  import CarsMiddleware from './cars_middleware';

const masterMiddleware = applyMiddleware(

$     CarsMiddleware
);

export default masterMiddleware;
```

Could be an easy fix? @JescaLyn

**Comment by JescaLyn** [comment]
​Yeah I think this issue is because of the difference between GNU sed
editor and Mac sed editor. GNU can handle tab literals, but Mac needs it to
be wrapped in a $'' ASCII quote tag.​ This might just need to be handled
with if/else logic about the kind of sed editor. I'll look into how to
determine which sed editor the user has

**Comment by paul-codes-1** [comment]
Resolved by #22
